deviceadvisor/tests/utils/datest_utils.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/crt/Types.h>
+#include <iostream>
+#include <stdlib.h>
+
+using namespace Aws::Crt;
+
+namespace DATest_Utils
+{
+
+    const char *ENV_ENDPONT = "DA_ENDPOINT";
```

Comment:
Maybe a separate class/structure (DeviceAdvisorEnvironment) with fields matching all of the possible values you want to read from the environment and an Init function or similar that reads everything in and initializes the fields appropriately?